### PR TITLE
server: support split transaction retry.

### PR DESCRIPTION
**Pull request by siddontang**
@qiuyesuifeng @ngaut

### CODE REVIEW

File: server/cluster.go [hunk 1]
```diff
@@ -324,6 +324,7 @@ func (s *Server) bootstrapCluster(clusterID uint64, req *pdpb.BootstrapRequest)
 	}
 	ops = append(ops, clientv3.OpPut(regionSearchPath, string(regionValue)))
 
+	// TODO: we must think an better to handle bootstrap failed, maybe intervene manually.
```
Comment (qiuyesuifeng) [inline server/cluster.go]: an better -> a better way?

File: server/util.go [hunk 2]
```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	requestTimeout = 3 * time.Second
+	requestTimeout = 30 * time.Second
```
Comment (qiuyesuifeng) [inline server/util.go]: Is it too long timeout?

Comment (siddontang) [inline server/util.go]: 3s is too short, can you think a better one?

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @ngaut

**Comment by ngaut** [comment]
LGTM

**Pull request merged**